Makefile

```diff
@@ -120,7 +120,7 @@ test-unit: envtest
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./controllers/... ./pkg/... ./rte/pkg/...
 
 test-e2e: build-e2e-all
-	hack/run-test-e2e.sh
+	ENABLE_SCHED_TESTS=true hack/run-test-e2e.sh
```

Comment:
ok for now, let's think about creating a separate lane

Reply:
The problem with a separate lane is limiting the sched tests to be independent of NRT objects, so we can test the scheduler installation but nothing more.

If we want to have access to NRT objects, we need the NROP CR installation which is exactly what we're getting here.

Reply:
fair point indeed. Let's keep it here for the time being.